src/custom/bluerov.rs

```diff
@@ -47,3 +47,49 @@ pub fn udp() -> Vec<VideoAndStreamInformation> {
         })
         .collect()
 }
+
+// pub fn rtsp() -> Vec<VideoAndStreamInformation> {
+//     video::video_source_local::VideoSourceLocal::cameras_available()
+//         .iter()
+//         .filter(|cam| {
+//             cam.inner()
+//                 .formats()
+//                 .iter()
+//                 .any(|format| format.encode == VideoEncodeType::H264)
+//         })
+//         .enumerate()
+//         .map(|(index, cam)| {
+//             let formats = cam.inner().formats();
+//             let format = formats
+//                 .iter()
+//                 .find(|format| format.encode == VideoEncodeType::H264)
+//                 .unwrap();
+
+//             // Get the biggest resolution possible
+//             let mut sizes = format.sizes.clone();
+//             sizes.sort_by(|first_size, second_size| {
+//                 (10 * first_size.width + first_size.height)
+//                     .cmp(&(10 * second_size.width + second_size.height))
+//             });
+//             let size = sizes.last().unwrap();
+
+//             dbg!(cam);
+
+//             VideoAndStreamInformation {
+//                 name: format!("RTSP Stream {}", index),
+//                 stream_information: StreamInformation {
+//                     endpoints: vec![
+//                         Url::parse(&format!("rtsp://localhost:{}/test", 8554 + index)).unwrap()
+//                     ],
+//                     configuration: CaptureConfiguration {
+//                         encode: format.encode.clone(),
+//                         height: size.height,
+//                         width: size.width,
+//                         frame_interval: size.intervals.first().unwrap().clone(),
+//                     },
+//                 },
+//                 video_source: cam.clone(),
+//             }
+//         })
+//         .collect()
+// }
```

Comment:
Just keeping it here to ask you if we'll gonna need something like this, 'cause I didn't get the reason why we are creating this default configuration for UDP.